src/web/mjs/connectors/KangaryuTeam.mjs

```diff
@@ -0,0 +1,65 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class KangaryuTeam extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'kangaryuteam';
+        super.label = 'Kangaryu Team';
+        this.tags = [ 'manga', 'french' ];
+        this.url = 'https://kangaryu-team.fr';
+    }
+
+    async _getMangas() {
+        let request = new Request(new URL('/directory', this.url), this.requestOptions);
+        let data = await this.fetchDOM(request, 'article#content div.group.center');
+
+        return data.map(element => {
+            return {
+                id: this.getRootRelativeOrAbsoluteLink(element.firstChild.href, this.url),
+                title: element.textContent.trim()
+            };
+        });
+    }
+
+    async _getChapters(manga) {
+        let url = new URL(manga.id, this.url);
+        let data = await this._fetchPOST(url, 'adult=true', 'article#content div.list div.element div.title > a');
+
+        return [...data].map(element => {
+            return {
+                id: this.getRootRelativeOrAbsoluteLink(element.href, this.url),
```

Comment:
May drop `.href` attribute, this is automatically used by `getRootRelativeOrAbsoluteLink`